charts/incubator/teslamate/templates/_configmap.tpl

```diff
@@ -0,0 +1,30 @@
+{{/* Define the configmap */}}
```

Comment:
This is so we dont have to manually list the dashboards. it will automatically pick them up in the folder and add them to configmaps.

They had to be split into multiple folder due to the 1MB limit on configmap sizes

Reply:
I would suggest splitting the dashboards into one configmap per file.
That makes it easier on deployment tools like flux/argo and allows you to use a single folder

Reply:
Done i split them into one config map per file